src/ui/Windows/PasswordSafe.rc

```diff
@@ -11,7 +11,8 @@
 #include "resource2.h"  // Menu, Toolbar & Accelerator resources
 #include "resource3.h"  // STRING resources
 #include "..\..\core\core.h"  // core STRING resources
-#include "VirtualKeyboard\VKresource.h"  // Virtual Keyboard resource

+#include "VirtualKeyboard\VKresource.h"  // Virtual Keyboard resource
+

```

Comment:
Since I didn't make any changes to this line, I don't understand why this shows up as changed content.